spreadsheet/row.go

```diff
@@ -98,7 +98,16 @@ func (r Row) AddCell() Cell {
 // to the slice will have no effect.
 func (r Row) Cells() []Cell {
 	ret := []Cell{}
+	lastIndex := -1
 	for _, c := range r.x.C {
+		ref, _ := reference.ParseCellReference(*c.RAttr)
```

Comment:
Should check error? output some debug message if problematic and handle ref as empty if problem?
Also may need to check with dereferencing pointer c.Attr
if c.RAttr == nil
Is there any guarantee that it is set?

Reply:
One way to think about this is whether a fuzzer would be able to provide an input that could crash the program. Our goal is to prevent that.

Reply:
Yes, it is possible to make it panic through manually changing cell.X().RAttr to nil. What to do in the case of error? just skip this cell or remove it from the sheet?

Reply:
Just skipping over it and outputing a debug message would be good in this case.

Reply:
fixed